### DISCUSSION TITLE
Intuitions on modifying training parameters

### DISCUSSION THREAD

**Discussion by jreus** [acids-ircam/RAVE#99] [General]
Hey all, I was thinking this would be a useful documentation post for RAVE users out there. 

Can you provide some intuition on when or why it is advisable to modify some of the more interesting training parameters? Does anyone have any experience modifying the following parameters outside of their defaults? Could you share why you changed these values and what the intended effects were? 

For example:
```
DATA_SIZE = 16

CAPACITY = setting(default=64, small=32, large=64)

LATENT_SIZE = 128

RATIOS = setting(
            default=[4, 4, 4, 2],
            small=[4, 4, 4, 2],
            large=[4, 4, 2, 2, 2],
        )

FEATURE_MATCH = True

LOUD_STRIDE = 1

USE_NOISE = True
NOISE_RATIOS = [4, 4, 4]
NOISE_BANDS = 5

D_CAPACITY = 16
D_MULTIPLIER = 4
D_N_LAYERS = 4

MODE = "hinge"

N_SIGNAL = 65536

BATCH = 8
```

**Comment by ghost** [comment]
sadly no answers on this. i was also looking into this in order to do very fast training on very small data sets, i wonder how i could do this.

**Comment by devstermarts** [comment]
I played around with LATENT_SIZE parameter when training with Wasserstein encoder because the default value of 128 resulted in models that I couldn't put to use (nn~ objects with the according number of in-/outlets which crashed PD on DSP processing). The new default of 16 worked just fine. 

At some point i'd need to understand or check in detail how many latent dimensions would make sense on which occasion (e.g. size and quality ("simple sounds" vs. full length tracks) of training data) - if anyone has recommendations or feedback on this matter, it'd be appreciated. 

One more question: when training full length tracks, would it be recommended to set a longer value on N_SIGNAL parameter to make sure a representative length of development within the piece is being processed?
